### PR TITLE
gpu: fix drawalpha "properly"

### PR DESCRIPTION
**Pull request by ThatGamerBlue**

### CODE REVIEW

File: src/main/java/com/openosrs/injector/injectors/raw/RasterizerAlpha.java [hunk 1]
```diff
@@ -125,27 +125,39 @@ public void inject() throws Injexception
 						StackContext alphaPop = isub.getPops().get(0);
 						InstructionContext alphaPusher = alphaPop.getPushed();
 						InstructionContext isubResult = isub.getPushes().get(0).getPopped().get(0);
+						boolean shouldSubtractLocal = false;
 
 						if (pushesToSameVar(isubResult, alphaPusher))
```
Comment (Lucwousin) [inline src/main/java/com/openosrs/injector/injectors/raw/RasterizerAlpha.java:130]: is pushesToSameVar side effect free? if so, comment that out as well

Comment (Lucwousin) [reply src/main/java/com/openosrs/injector/injectors/raw/RasterizerAlpha.java:130]: nvm

Comment (ThatGamerBlue) [reply src/main/java/com/openosrs/injector/injectors/raw/RasterizerAlpha.java:130]: ok

**Comment by ThatGamerBlue** [comment]
fuck it

**Pull request merged**